components/script/dom/permissions.rs

```diff
@@ -322,10 +322,25 @@ pub fn get_descriptor_permission_state(
                 .permission_state_invocation_results()
                 .borrow_mut()
                 .remove(&permission_name.to_string());
-            prompt_user(&format!(
-                "The {} {}",
-                permission_name, NONSECURE_DIALOG_MESSAGE
-            ))
+
+            prompt_user(
+<<<<<<< HEAD
+<<<<<<< HEAD
+                &format!(
+                    "The {} {}",
+                    permission_name, 
+                    NONSECURE_DIALOG_MESSAGE
+                ), 
+                settings_window.is_headless()
+=======
+                &format!("The {} {}", permission_name, NONSECURE_DIALOG_MESSAGE),
+                settings.is_headless(),
+>>>>>>> cbd25f58b3... fixup! Removed opts::get() from script/dom/permissions.rs
+=======
+                &format!("The {} {}", permission_name, NONSECURE_DIALOG_MESSAGE),
+                settings_window.is_headless(),
+>>>>>>> c7b6054db3... Removed opts::get() from script/dom/characterdata.rs
```

Comment:
Whoops 👀 Tidy is complaining about this

Reply:
Yep, somehow missed those conflict indicators. Will force-push now.